src/Compilers/CSharp/csc/csc.csproj

```diff
@@ -11,8 +11,9 @@
     <Prefer32Bit>false</Prefer32Bit>
     <LargeAddressAware>true</LargeAddressAware>
     <StartupObject>Microsoft.CodeAnalysis.CSharp.CommandLine.Program</StartupObject>
-    <TargetFramework>net46</TargetFramework>
-    <RuntimeIdentifiers>win7</RuntimeIdentifiers>
+    <TargetFrameworks>net46;netcoreapp2.0</TargetFrameworks>
```

Comment:
This should be a variable - `RoslynPortableTargetFrameworks46` in `build/Targets/Settings.props`. (Reason I'm leaving like that this is so I can check exactly what we should be targeting - iirc I heard something about net461 (VBCSCompiler is net46), but I'm not sure)